event-management-backend/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM maven:3.8.1-openjdk-11 as build
+WORKDIR /build
+COPY pom.xml .
+RUN mvn dependency:go-offline
+
+COPY src/ /build/src/
+RUN mvn clean package -DskipTests spring-boot:repackage
+
+FROM openjdk:11-jre
+COPY --from=build /build/target/*.jar /app/*.jar
+CMD java -jar /app/*.jar
```

Comment:
There are 2 ways to run a command: shell (used here) and exec. The difference is that running the command in exec mode enables the app to listen to system signals (e.g. SIGTERM). More info [here](https://spring.io/guides/topicals/spring-boot-docker/) if you scroll below.

Reply:
Changed.